_events/2020-01-20-GalaxyWS_scrna_FR.md

```diff
@@ -0,0 +1,127 @@
+---
+site: freiburg
+tags:
+- training
+title: Galaxy single cell RNA-seq data analysis workshop in Freiburg
+starts: 2020-03-23
+ends: 2020-03-27
+organiser:
+  name: Freiburg Galaxy Team & MPI Freiburg
+  email: contact@usegalaxy.eu
+location:
+  name: PC Pool 3
+  street: Werthmannstraße 4 
+  postal: 79098
+  city: Freiburg im Breisgau
+  region: Baden-Württemberg
+  country: Germany
+  geo:
+    lat: 47.993469
+    lon: 7.845275
+supporters:
+- unifreiburg
+- denbi
+- mpi
+# special hiding of footer since we want to do it manually.
+hidefooter: true
+---
+
+We are offering a Galaxy workshop on single cell data analysis. 39 participants will be selected from all applicants. 
+To apply for attendance, please sign in [here](https://forms.gle/uutHkdgsJUgKnHhf7).
+
+## Venue
+
+{% include map.html location=page.location showmap=true zoomlevel=15 hidepopup=true %}
+
+
+# Important notes
+
+1. If you are registered but **cannot attend** our workshop, please [contact us
+   by email](mailto:contact@usegalaxy.eu) immediately. We have a long waiting
+   list and can give your place to others, even on short notice.
+2. Please register to our [European Galaxy server](https://usegalaxy.eu) to perform the analysis. Participants from the     
+   MPI Freiburg, please note that the European Galaxy server is different from the MPI Galaxy server. You will need to 
+   have an account on [https://usegalaxy.eu](https://usegalaxy.eu]) as well.
+3. You can bring your *own notebook* or desktop computers will be available. Eduroam is available, ask your institute 
+   for how to login.
+4. The workshop is free of charge. No stipends for travel or accommodation are available.
+
+## Program:
+
+Every day the workshop will run from 9:00-17:00 (give or take, depending on questions at the end). If the times will 
+change, it will be announced during the workshop.
+
+Preliminary schedule. Topics may be adjusted according to the participants demand
+
+| Day | Topics                                                                         |
+|:---:|:-------------------------------------------------------------------------------|
+| Mon | Pre-processing of scRNA-seq data and Barcoding                                 |
```

Comment:
Do you assume people can use Galaxy already? No 101 at the beginning?
Do you really need  6h for every of these tutorials?

Reply:
Is the 101 performed for every workshop? Do the RNA-seq trainings have the 101 on the first day? If so, I am happy to add

Reply:
I guess what I'm asking is: If they do the Galaxy 101, will they then need to do the RNA-seq tutorial, and only *then* they can do the single cell stuff?

Reply:
Yes, I think so. But the question is not what other workshops did, rather what do you expect from your participants ... should they know Galaxy already, or can people join that have never seen Galaxy.

Reply:
Okay, how about monday is Galaxy 101 and Mapping, tuesday is general RNA-seq, and then wed/thu/fri is scRNA-seq ?

https://github.com/usegalaxy-eu/website/pull/380/commits/08b03c4057f1f8f8b1a2c6e0e2aa6e4d07c80e26